docs_src/source/user_guide.md

```diff
@@ -0,0 +1,91 @@
+# User Guide
+
+A general overview of the motivation for and functionality of Plot Curve, as implemented in
+**seeq-plot-curve** is provided in this section.
+
+## The Problem
+
+When working with time series data, it is very common to want to create a new signal based on a relationship or function with 
+an existing signal.  When the  formula for the calculation is simple and known, it can be directly input into Seeq 
+via the formula editor.  But many relationships are empirical in nature, represented by data points which describe the 
+relationship between variables.  
+
+Some examples may include :
+
+- Equipment Curves 
+  - A [pump curve](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pumpsandsystems.com%2Funderstanding-pump-curves&psig=AOvVaw1vTS_8NFaEJZ1ICnwKBoN3&ust=1643391727854000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCKD-hrO90vUCFQAAAAAdAAAAABAD) providing the relationships between flow, head, efficiency, etc
+  - A [valve curve](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.industrialcontrolsonline.com%2Ftraining%2Fonline%2Fcontrolling-flow-ball-valves&psig=AOvVaw0_zt9UE6iAyvddeIh4UnAJ&ust=1643391749751000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCOCw6r290vUCFQAAAAAdAAAAABAF) describing the relationship between the flow coefficient of a valve and its opening percentage
```

Comment:
These google links serve a "Redirect Notice" in Chrome.  I think that replacing them with the final destination should take care of that.